getting-started/000-getting-started.md

```diff
@@ -1,16 +1,20 @@
 Get started with Mapzen Search
 
-Mapzen Search is a modern geocoding and place search service based on [Pelias](https://github.com/pelias/pelias), an experimental open geocoder from [Mapzen](https://mapzen.com) and powered by open data. It's used to transform the way people talk about places - with names and addresses - into geographic coordinates.
+Mapzen Search is a modern geocoding and place search service based on [Pelias](https://github.com/pelias/pelias), an experimental, open geocoder from [Mapzen](https://mapzen.com) and powered by open data. It's used to transform the way people talk about places - with names and addresses - into geographic coordinates, and to translate geographic coordinates into recognizable names of places.
 
-[REWRITE]
 Mapzen Search can be used to:
 - geocode addresses, points of interest, neighbourhood, cities, and regions
 - power real-time search autocomplete for places
-- turn a list of addresses into coordinates to plot
+- turn a list of addresses into coordinates that can be plotted on a map or computed
 - Take location coordinates and turn them into human readable places, like an address
 - find the neighbourhood, cities, states, and countries of location coordinates to quickly classify where your data is coming from
```

Comment:
so I guess the idea here is the top bunch of bullets is the "high level problems we can solve for you" and the bottom bullets are "all the features we provide"?